### PR TITLE
Allow user to send app logs 

### PR DESCRIPTION
**Pull request by Bluesir9**
Added option to send app logs in Settings menu. Will take care of #736. 

PFA screenshot below:

screenshot_1508580320

### CODE REVIEW

**Comment by nicolas-raoul** [comment]
Thanks a lot!
May I suggest using "log" rather than "logs"? Singular rather than plural
We are talking about the app's log file. Using plural could imply than we collect more than that, which can sound scary.

**Comment by misaochan** [comment]
> May I suggest using "log" rather than "logs"? Singular rather than plural
We are talking about the app's log file. Using plural could imply than we collect more than that, which can sound scary.

If "logs" doesn't sound right, perhaps "send log file"? I've never seen "log" used as a singular noun by itself in the tech context, haha.

**Comment by Bluesir9** [comment]
Have changed usage of "logs" to "log file" in both the main title and description as requested.

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -207,5 +207,7 @@ Tap this message (or hit back) to skip this step.</string>
   <string name="give_permission">Give permission</string>
   <string name="use_external_storage">Use external storage</string>
   <string name="use_external_storage_summary">Save pictures taken with the in-app camera on your device</string>
-    <string name="login_to_your_account">Login to your account</string>
+  <string name="send_log_file">Send Log file</string>
```
Comment (nicolas-raoul) [inline app/src/main/res/values/strings.xml]: Sorry for the tiny detail, but how about "Send log file"? :-)

Comment (Bluesir9) [reply app/src/main/res/values/strings.xml]: Ha ha, that capital L was bothering me too! Go ahead and review the rest. Once you are done I will make all the requested changes together and push them.

**Comment by misaochan** [comment]
Thanks for submitting this @Bluesir9 , we will try and test it soon. :) BTW, would you like to be added to our credits.md file? If so, what name would you prefer?

Cheers!

**Comment by Bluesir9** [comment]
@misaochan That would be nice! :). You can use my real name "Vishan Seru" for that. Thanks 👍

**Comment by misaochan** [comment]
@Bluesir9 Done :)

**Comment by neslihanturan** [comment]
Tested it and it works, thanks @Bluesir9 :)

**Pull request merged**